src/rpc/rawtransaction.cpp

```diff
@@ -438,13 +438,14 @@ static void getassetunlockstatuses_help(const JSONRPCRequest& request)
 {
     RPCHelpMan{
             "getassetunlockstatuses",
-            "\nReturns the status of given Asset Unlock indexes.\n",
+            "\nReturns the status of given Asset Unlock indexes. Block height can be also specified\n",
```

Comment:
maybe smth like
```suggestion
            "\nReturns the status of given Asset Unlock indexes at the tip of the chain or at a specific block height if specified.\n",
```
?
@thephez

Reply:
LGTM :+1: